src/reconcile/rec_visibility.c

```diff
@@ -248,6 +248,75 @@ __timestamp_no_ts_fix(WT_SESSION_IMPL *session, WT_TIME_WINDOW *select_tw)
     return (false);
 }
 
+/*
+ * __rec_delete_hs_upd --
```

Comment:
This seems to belong in src/history/hs_rec.c - or am I missing something?

Reply:
We can move it there.

Reply:
Fixed in [ffea256](https://github.com/wiredtiger/wiredtiger/pull/7939/commits/ffea25641f7d62ac1919f643ba0d12ae1b3c2075)